### PR TITLE
Update common-errors.md

### PR DESCRIPTION
**Pull request by ziqi-hu**
Add common-errors about Dargs.
Signed-off-by: ziqi-hu <51811006+ziqi-hu@users.noreply.github.com>

### CODE REVIEW

File: doc/troubleshooting/common-errors.md [hunk 1]
```diff
@@ -28,4 +28,7 @@ Some common reasons are as follows:
 2. You may have something wrong in your input files, which causes the process to fail.
 
 ## RuntimeError: find too many unsuccessfully terminated jobs.
-The ratio of failed jobs is larger than ratio_failure. You can set a high value for ratio_failure or check if there is something wrong with your input files. 
\ No newline at end of file
+The ratio of failed jobs is larger than ratio_failure. You can set a high value for ratio_failure or check if there is something wrong with your input files. 
+
+## Dargs: xxx is not allowed in strict mode.
+Strict format check has been applied since version 0.10.6.  To avoid misleading users, some older-version keys that are already ignored or absorbed into default settings are not allowed to be present. And the expected structure of the dictionary in the param.json also differs from those before versions 0.10.6. This error will occur when format check finds older-fashion keys in the json file.  Please try deleting or annotating these keys, or correspondingly modulate the json file. Example files in the newest format could be found in [examples](https://github.com/deepmodeling/dpgen/tree/master/examples).
```
Comment (wanghan-iapcm) [inline doc/troubleshooting/common-errors.md:34]: Should be `0.10.7` published in the near future.

**Review by HuangJiameng — APPROVED** [review]

File: doc/troubleshooting/common-errors.md (hunk 1 above)
Comment (njzjz) [reply doc/troubleshooting/common-errors.md:34]: We have not add the check yet.

Comment (njzjz) [reply doc/troubleshooting/common-errors.md:34]: At present I only add strict check to init_reaction.

**Comment by njzjz** [comment]
Please set a proper PR title.

**Pull request merged**